conf/ansible_hub.yaml.template

```diff
@@ -0,0 +1,4 @@
+ANSIBLE_HUB:
```

Comment:
Please define a validator for this namespace:

https://github.com/SatelliteQE/robottelo/blob/master/robottelo/config/validators.py